src/js/actions/ChapterReviewActions.js

```diff
@@ -0,0 +1,197 @@
+import axios from 'axios';
+import config from '../../config/config';
+
+export const getAlternateTakes = (selectedTakes) => {  // chunkNum comes from the NavBar to display on the comments section the chunk number
+  return function(dispatch) {
+
+    selectedTakes.forEach((take) => {
+      return axios
+        .get(`${config.apiUrl}takes/?chunk_id=${take.publishedTake.chunk}`,
+          {
+            headers: { Authorization: 'Token ' + localStorage.getItem('token') },
+          })
+        .then(response => {
+          let takes = response.data;
+          processTakes(takes).then( (tempArray) => {
+            dispatch(getAlternateTakesSuccess(tempArray));
+          });
+        })
+        .catch(error => {
+          console.log(error);
+        });
+    });
+  };
+};
+
+async function processTakes(takes) {
+  let tempArray =[];
+  let chunkId ='';
+  for (const take of takes) {
+    if (take.rating ===3 && take.published === false) {
+      tempArray.push(take);
+      chunkId= take.chunk;
+    }
+  }
+  return {chunkId: chunkId, takes: tempArray};
+}
+
+export const getAlternateTakesSuccess = (alternateTakes) => {
+  return {
+    type: 'FETCH_ALTERNATE_TAKES',
+    alternateTakes,
+  };
+};
+
+// export function playTake(takeId) {
+//   return {
+//     type: 'UPDATE_PLAYING_TAKE',
+//     takeId,
+//   };
+// }
```

Comment:
is this going to be useful in the future? or just forgot to delete it?

Reply:
yes just forgot to delete, i'll remove it